pygmt/tests/test_timestamp.py

```diff
@@ -93,38 +93,3 @@ def test_timestamp_text_truncated():
         fig.timestamp(text="0123456789" * 7)
         assert len(record) == 1  # check that only one warning was raised
     return fig
-
-
-@pytest.mark.mpl_image_compare(filename="test_timestamp.png")
-def test_timestamp_deprecated_timestamp(faketime):
-    """
-    Check if the deprecated parameter 'timestamp' works but raises a warning.
-    """
-    fig = Figure()
-    with pytest.warns(expected_warning=SyntaxWarning) as record:
-        with config(FORMAT_TIME_STAMP=faketime):
-            # plot nothing (the data is outside the region) but a timestamp
-            fig.plot(
-                x=0,
-                y=0,
-                style="p",
-                projection="X1c",
-                region=[1, 2, 1, 2],
-                timestamp=True,
-            )
-        assert len(record) == 1  # check that only one warning was raised
-    return fig
-
-
-@pytest.mark.mpl_image_compare(filename="test_timestamp.png")
-def test_timestamp_deprecated_u(faketime):
-    """
-    Check if the deprecated parameter 'U' works but raises a warning.
-    """
-    fig = Figure()
-    with pytest.warns(expected_warning=SyntaxWarning) as record:
-        with config(FORMAT_TIME_STAMP=faketime):
-            # plot nothing (the data is outside the region) but a timestamp
-            fig.plot(x=0, y=0, style="p", projection="X1c", region=[1, 2, 1, 2], U=True)
-        assert len(record) == 1  # check that only one warning was raised
-    return fig
```

Comment:
Need to modify this test to check that the GMTInvalidInput exception is raised if `U` is used.

Reply:
See https://github.com/GenericMappingTools/pygmt/pull/3045/files#r1502738446.